modules/post/windows/gather/credentials/securecrt.rb

```diff
@@ -159,10 +165,13 @@ def run
     parent_key = 'HKEY_CURRENT_USER\\Software\\VanDyke\\SecureCRT'
     # get session file path
     securecrt_path = expand_path(registry_getvaldata(parent_key, 'Config Path') + session.fs.file.separator + 'Sessions')
-    unless securecrt_path.to_s.empty?
+    if securecrt_path.to_s.empty?
+      print_error('Could not find the registry entry for the SecureCRT session path. Ensure that SecureCRT is installed on the target.')
+    else
```

Comment:
Technically this can be done either way, but I do agree that having the exception case first followed by the actual work we want to do does look neater 👍